src/EnergyPlus/CMakeLists.txt

```diff
@@ -979,7 +979,7 @@ if(LINK_WITH_PYTHON)
       TARGET energyplusapi
       POST_BUILD # TODO: I don't think we want to quote the generator expression
       COMMAND ${CMAKE_COMMAND} -E env --unset=PIP_REQUIRE_VIRTUALENV
-      ${Python_EXECUTABLE} -m pip install --target="$<TARGET_FILE_DIR:energyplusapi>/python_lib" --upgrade energyplus-launch==3.7.2
+      ${Python_EXECUTABLE} -m pip install --target="$<TARGET_FILE_DIR:energyplusapi>/python_lib" --upgrade energyplus-launch==3.7.2 energyplus-transition-tools==2.1.1
```

Comment:
Just grab the transition tool package, at version 2.1.1 which I just released this morning after some cleanups.

Reply:
When do we start using https://github.com/Myoldmopar/EnergyPlusPythonApps/ as a meta package itself?

Reply:
* https://github.com/Myoldmopar/EnergyPlusPythonApps/pull/6